packages/tslint-config/blueprint-rules.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+/**
+ * Enable Blueprint-specific TSLint rules defined in this package.
```

Comment:
@JKillian @rcchen new top-level file in this package that configures blueprint-specific rules.